controllers/vrg_volrep.go

```diff
@@ -633,6 +633,8 @@ func (v *VRGInstance) UploadPVandPVCtoS3Store(s3ProfileName string, pvc *corev1.
 func (v *VRGInstance) UploadPVAndPVCtoS3(s3ProfileName string, objectStore ObjectStorer,
 	pv *corev1.PersistentVolume, pvc *corev1.PersistentVolumeClaim,
 ) error {
+	v.log.Info("About to upload PV to s3 profile", "PV", pv.Name, "s3 profile", s3ProfileName)
+
 	if err := UploadPV(objectStore, v.s3KeyPrefix(), pv.Name, *pv); err != nil {
```

Comment:
We can log inside UploadPV - this way all callers get logging and we cannot forget to log.